internal/otelcollector/config/metricagent/kubeletstats_receivers_test.go

```diff
@@ -22,24 +23,15 @@ const (
 	none        metricResource = "none"
 )
 
-func TestReceivers(t *testing.T) {
+func TestKubeletStatsReceiver(t *testing.T) {
```

Comment:
was there a reason to have a `kubeletstats_receiver_test.go` instead of depending on the golden test files used by the `config_builder_test.go`?

Reply:
I decided to keep them since both receiver configuration building was quite complex and the unit test sets were quite comprehensive (comparing to our typical copy/pasted tests)